ui/StatusQ/src/StatusQ/Core/Theme/Theme.qml

```diff
@@ -16,58 +15,210 @@ QtObject {
+        }
+
+        property var codeFontMedium: FontLoader {
+            source: sqAssetPath + "fonts/RobotoMono/RobotoMono-Medium.ttf"
+        }
+
+        property var codeFontBold: FontLoader {
+            source: sqAssetPath + "fonts/RobotoMono/RobotoMono-Bold.ttf"
+        }
+    }
+
+    property int secondaryAdditionalTextSize: 17
     property int primaryTextFontSize: 15
     property int secondaryTextFontSize: 14
+    property int additionalTextSize: 13
     property int tertiaryTextFontSize: 12
     property int asideTextFontSize: 10
 
-    function setTheme(theme) {
-        palette = theme
-    }
+    property int xlPadding: 32
+    property int bigPadding: 24
+    property int padding: 16
+    property int halfPadding: 8
+    property int smallPadding: 10
+    property int radius: 8
 
-    function updateFontSize(fontSize) {
+    function updateFontSize(fontSize:int) {
         switch (fontSize) {
             case Theme.FontSizeXS:
+                secondaryAdditionalTextSize = 15
                 primaryTextFontSize = 13
                 secondaryTextFontSize = 12
+                additionalTextSize = 11
                 tertiaryTextFontSize = 10
                 asideTextFontSize = 8
                 break;
 
             case Theme.FontSizeS:
+                secondaryAdditionalTextSize = 16
                 primaryTextFontSize = 14
                 secondaryTextFontSize = 13
+                additionalTextSize = 12
                 tertiaryTextFontSize = 11
                 asideTextFontSize = 9
                 break;
 
             case Theme.FontSizeM:
+                secondaryAdditionalTextSize = 17
                 primaryTextFontSize = 15
                 secondaryTextFontSize = 14
+                additionalTextSize = 13
                 tertiaryTextFontSize = 12
                 asideTextFontSize = 10
                 break;
 
             case Theme.FontSizeL:
+                secondaryAdditionalTextSize = 18
                 primaryTextFontSize = 16
                 secondaryTextFontSize = 15
+                additionalTextSize = 14
                 tertiaryTextFontSize = 13
                 asideTextFontSize = 11
                 break;
 
             case Theme.FontSizeXL:
+                secondaryAdditionalTextSize = 19
                 primaryTextFontSize = 17
                 secondaryTextFontSize = 16
+                additionalTextSize = 15
                 tertiaryTextFontSize = 14
                 asideTextFontSize = 12
                 break;
 
             case Theme.FontSizeXXL:
+                secondaryAdditionalTextSize = 20
                 primaryTextFontSize = 18
                 secondaryTextFontSize = 17
+                additionalTextSize = 16
                 tertiaryTextFontSize = 15
                 asideTextFontSize = 13
                 break;
         }
     }
+
+    // Style compat
+    readonly property string assetPath: Qt.resolvedUrl("../../../../../imports/assets/")
```

Comment:
This is realyl not cool because we access stuff from out of the project (StatusQ)... Maybe we should move them to StatusQ? :thinking:

Reply:
Yeah not fond of it too much... another PR/issue maybe? I don't feel like touching this anymore, it works for now but definitely sth to fix next

Reply:
@micieslak alright, added as a second commit: https://github.com/status-im/status-desktop/pull/16527/commits/e3b7df4f366b05718b51d2d73eb7efb0e8fb4a18 :relieved: